transformers/shared/constants/signature.go

```diff
@@ -32,9 +32,11 @@ func JugFileIlkSignature() string         { return getLogNoteTopicZero(jugFileIl
 func JugFileVowSignature() string         { return getLogNoteTopicZero(jugFileVowMethod()) }
 func JugInitSignature() string            { return getLogNoteTopicZero(jugInitMethod()) }
 func LogBumpSignature() string            { return getEventTopicZero(logBumpEvent()) }
+func LogBuyEnabledSignature() string      { return getEventTopicZero(logBuyEnabledEvent()) }
 func LogItemUpdateSignature() string      { return getEventTopicZero(logItemUpdateEvent()) }
 func LogKillSignature() string            { return getEventTopicZero(logKillEvent()) }
 func LogMakeSignature() string            { return getEventTopicZero(logMakeEvent()) }
+func LogMatchingEnabledSignature() string { return getEventTopicZero(logMatchingEnabledEvent()) }
```

Comment:
What's up with the added `LogMatchingEnabled` stuff?

Reply:
I think I was anticipating doing those all in one PR so I just did the work at the same time with respect to signatures.